porcupine/menubar.py

```diff
@@ -449,3 +449,7 @@ def add_link(menu_path: str, label: str, url: str) -> None:
         "https://github.com/Akuli/python-tutorial/blob/master/README.md",
     )
     add_link("Help/Python", "Official documentation", "https://docs.python.org/")
+
+    def setup() -> None:
+        pluginmanager.get_menu("Settings").add_command(label="Plugin Manager", command=show_dialog)
```

Comment:
This shouldn't be in a `setup()` function anymore. It should look very similar to this line above that creates the "Porcupine Settings" dialog:

```python3
get_menu("Settings").add_command(label="Porcupine Settings", command=settings.show_dialog)
```

and probably be just after that line, so that it's nicely in the same place with other related code.